requirements.txt

```diff
@@ -58,3 +58,6 @@ unicodecsv==0.14.1,<1.0
 httplib2~=0.9.2
 unidecode
 
+# Db2 client
+ibm_db
```

Comment:
could you put the deps in https://github.com/lyft/amundsendatabuilder/blob/master/setup.py#L51 as extra required? thanks.